src/tests/EdFi.DataManagementService.Tests.E2E/Features/References/NaturalForeignKeyValidation.feature.disabled

```diff
@@ -0,0 +1,189 @@
+Feature: Validation of Natural and Foreign Keys
+
+        Background
+            Given the system has this School
```

Comment:
I think you have to name this "Given the system has these schools" for the binding to work even though there is only one school. We don't have any binding with the word "this".

Reply:
Changed